Shop-Facade-Open-Api-Spec.yaml

```diff
@@ -0,0 +1,1104 @@
+      operationId: getS100Permits
+      parameters:
+      - $ref: "#/components/parameters/licenceId"
+      responses:
+        "200":
+          description: OK - Returns a zip containing permit files.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+            Content-Disposition:
+              schema:
+                type: string
+                format: filename=permits.zip; filename*=UTF-8''Permits.zip
+          content:
+            application/zip:
+              schema:
+                title: Zip containing S-100 permit files.
+                type: string
+                format: binary
+        "204":
+          description: No Content - There are no S-100 permits for the requested licence.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+        "400":
+          description: Bad request - could be missing or invalid licenceId, it must be an integer and greater than zero.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/errorDescription'
+              example:
+                correlationId: 6c254109-8a0f-4446-8649-721775ba9de2
+                errors:
+                - source: licenceId
+                  description: Invalid licenceId
+        "401":
+          description: Unauthorised - either you have not provided valid token, or your token is not recognised.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+        "403":
+          description: Forbidden - you have no permission to use this API.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+        "404":
+          description: Licence not found
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/errorDescription'
+              example:
+                correlationId: 6c254109-8a0f-4446-8649-721775ba9de2
+                errors:
+                - source: licenceId
+                  description: Licence not found
+            application/zip:
+              schema:
+                $ref: '#/components/schemas/errorDescription'
+        "429":
+          description: Too Many Requests.
```

Comment:
Would we implement any throttling for Shop Facade Service and hence is it needed?